src/arch/x86/x86_64/src/consts/mod.rs

```diff
@@ -0,0 +1,11 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+pub mod msr;
+pub mod x86;
+
+#[allow(non_upper_case_globals)]
```

Comment:
what's a better place for these names.

Reply:
Maybe a new crate in Oreboot/src/lib ?
We only have device_tree and print in there

Reply:
There's also this existing crate: https://docs.rs/kib/4.0.0/kib/

Reply:
I'd rather not do a full dependency for a few constants. I'll put it in src/lib